client/additional-methods-setup/upe-preview-methods-selector/index.scss

```diff
@@ -0,0 +1,4 @@
+.upe-preview-methods-selector {
+	max-width: 680px;
+	margin: 0 auto;
+}
```

Comment:
As far as I know, the future state of the app will only include one flow, so that's why I just ended up copying it - just so that it's encapsulated in a directory that can easily be removed without too many external dependencies 😄

Reply:
OIC. Makes sense!